proc_dash/utility.py

```diff
@@ -44,28 +54,56 @@ def get_required_bagel_columns() -> list:
 
 # TODO: When possible values per column have been finalized (waiting on mr_proc),
 # validate that each column only has acceptable values
-def get_missing_required_columns(bagel: pd.DataFrame) -> set:
+def get_missing_required_columns(bagel: pd.DataFrame, schema_file: str) -> set:
     """Identifies any missing required columns in bagel schema."""
-    missing_req_columns = set(get_required_bagel_columns()).difference(
-        bagel.columns
-    )
+    missing_req_columns = set(
+        get_required_bagel_columns(schema_file)
+    ).difference(bagel.columns)
 
     return missing_req_columns
```

Comment:
Why not return directly - would be cleaner to not first create `missing_req_columns` since it isn't used for anything else